### PR TITLE
Rework `TraverseSyncTo...#handle...` methods to return void instead of boolean

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToParent.java [hunk 1]
```diff
@@ -26,8 +27,9 @@
 import com.virtuslab.gitmachete.frontend.vfsutils.GitVfsUtils;
 import com.virtuslab.qual.async.ContinuesInBackground;
 
-@ExtensionMethod({GitMacheteBundle.class, GitVfsUtils.class})
 @CustomLog
+@ExtensionMethod({GitMacheteBundle.class, GitVfsUtils.class})
+@SuppressWarnings("MissingSwitchDefault")
```
Comment (mkondratek) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToParent.java:32]: what happens when the value does not match any of the declared cases?

**Review by mkondratek — APPROVED** [review]
_widziałem to przychodzące_

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToParent.java (hunk 1 above)
Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToParent.java:32]: Just nothing... traverse stops (as we've fallen into `com.intellij.openapi.ui.MessageConstants#CANCEL`)

**Review by mkondratek — APPROVED** [review]
ok!

**Pull request merged**